tools/unrmdp.cpp

```diff
@@ -43,25 +46,30 @@ int main(int argc, char** argv) {
 	app.add_option("rmdpfile", rmdpFile, "The rmdp file containing the archives raw data")
 			->check(CLI::ExistingFile)
 			->required();
-	
+
 	app.add_flag("-l, --list", onlyListFiles, "List files in an archive without actually extracting them");
 
 	CLI11_PARSE(app, argc, argv);
 
+	fmt::print("Processing index...\n");
+
 	AWE::RMDPArchive rmdp(
 		new Common::ReadFile(binFile),
 		new Common::ReadFile(rmdpFile)
 	);
 
 	for (size_t i = 0; i < rmdp.getNumResources(); ++i) {
 		const std::string path = rmdp.getResourcePath(i);
+		const std::string normalizedPath = AWE::getNormalizedPath(path);
```

Comment:
Do we really need to handle normalized paths here? The whole point of the unrmdp tool is to unpack the data exactly as it was packed. From my perspective the changes in resman and rmdparchive would have been enough

Reply:
The main goal here was to remove the "d:/data" prefix of the path. The d: is not portable. Before this PR this was done in `RMDPArchive::getResourcePath` through `Common::replace(path, "d:/data/", "");` but prefix handling does not really belong there anymore. I used this function to remove the prefix to avoid having to list explicit prefixes in the unrmdp (and because it was available)...

Reply:
You are right about the fact that d: does not work, I was not aware of this. Maybe you could add an option to unpack the complete path for systems and add an exception if create_directories returns false?

Reply:
@Nostritius Such option means that you loose in a way control where the files will end up on your system. The root folder old drive letter may not always be suitable. It would be good if we can keep the files relative to the current working folder.
Would you agree if I only remove the prefixed <drive letter>: and / for now?
In a next step, the removed drive letter could be saved in a small metadata file which can then be picked up by a (eventual future) rmdp tool to fully reconstruct the data as it was...
When files are listed with --list or during unpacking it may indeed be good to show the full path...

Reply:
What about prefixing the result path with a "./" to explicitly state that the folder should be created relative to the working directory? You can do this for both, the normalized and the full path scenario.

Reply:
Sorry for the delay. I changed the code to remove only the drive prefix. Explicitly adding ./ was not needed because the unrmdp archives never start with a '/'. 
unrmdp is now showing both the original path and the extract path so that the destination is clear when there was a (small) change to the path.